connectors/routing/config.go

```diff
@@ -100,7 +100,7 @@ type Config struct {
 }
 
 // FindRouter finds the router information based on scope and namePrefix.
-func (c *Config) FindRouter(scope, namePrefix string) (*Rule) {
+func (c *Config) FindRouter(scope, namePrefix string) *Rule {
```

Comment:
this belongs in a different commit if you intend to change this (probably not a bad idea)